tensorflow_hub/resolver.py

```diff
@@ -518,6 +520,14 @@ def _set_url_context(self, context):
 
   def _call_urlopen(self, request):
```

Comment:
Let's move this logic into a helper and call it from __init__() and _set_url_context() instead, e.g.

```
def _maybe_disable_cert_validation(self):
  if os.getenv(_TFHUB_DISABLE_CERT_VALIDATION, ""):
      if self._context is None:
        self._context = ssl.create_default_context()
      self._context.check_hostname = False
      self._context.verify_mode = ssl.CERT_NONE


def __init__(self):
    self._context = None
    self._maybe_disable_cert_validation()

def _set_url_context(self, context):
    self._context = context
    self._maybe_disable_cert_validation()
```

Reply:
Let's actually remove the code on lines 525 - 530